openlibrary/templates/book_providers/cita_press_read_button.html

```diff
@@ -1,21 +1,11 @@
-$def with(cita_press_id)
+$def with(edition_key, cita_press_id)
 
 <div class="cta-button-group">
-  <a
-      href="https://citapress.org/$cita_press_id"
+  <a href="/books/${edition_key}/-/borrow?action=read"
       title="$_('Read eBook from Cita Press')"
       class="cta-btn cta-btn--available cta-btn--read cta-btn--external cta-btn--cita-press"
       target="_blank"
-       aria-haspopup="true"
-       aria-controls="cita-press-toast"
+      aria-haspopup="true"
+      aria-controls="cita-press-toast"
```

Comment:
Since these buttons no longer trigger toast messages, `aria-haspopup` and `aria-controls` can be removed.

Reply:
This applies to all affected direct provider buttons.